.github/workflows/audit_tracker.yml

```diff
@@ -8,6 +8,7 @@ jobs:
     uses: ./.github/workflows/_ruby-package.yml
     with:
       package: ${{ github.workflow }}
-      ruby: '["2.7.4", "3.1.2"]'
-      rails: '["7.0.3.1","6.1.6.1","6.0.5.1","5.2.8.1"]'
+      workdir: 'packages/${{ github.workflow }}'
+      ruby: '["2.7", "3.1"]'
+      rails: '["7.0","6.1","6.0","5.2"]'
```

Comment:
These all have to be copied, or can they share this config? Assuming they would always be kept in sync.

Reply:
@wadewinningham I can make these a default, but right now each package has a chance to configure what they support. The shared workflow provides the compatibility table so incompatible versions don't run in a build.